checks.d/etcd.py

```diff
@@ -173,6 +175,11 @@ def _get_json(self, url, ssl_params, timeout):
                                message="Timeout when hitting %s" % url,
                                tags=["url:{0}".format(url)])
             raise
+        except:
```

Comment:
Could you catch the `Exception` class here instead? It doesn't make sense to catch `KeyboardInterrupt`s here for instance.